projects/plugins/crm/readme.txt

```diff
@@ -12,230 +12,193 @@ The CRM for small businesses. Manage leads, invoicing, billing, email marketing,
+The Jetpack CRM plugin installs in seconds, with a super-quick welcome wizard to get you started. Adding a contact is straightforward. Creating a quote, or invoice: even simpler. Give it a go, you'll see!
 
-<blockquote>
-= ⭐ Mastering Jetpack CRM =
-1. Install this WordPress CRM plugin
-2. Step through the guided Welcome Wizard
-3. Add your first contact (lead or customer)
-</blockquote>
+Easy CRMs get used. Easy CRMs make your contact data work for you. We aim to make this easy, so you can get on with running your business.
 
-Easy CRM's get used. Easy CRM's make your contact data work for you. We aim to make this easy, so you can get on with running your business.
+Need a helping hand? No worries. Just [ask here](https://kb.jetpackcrm.com/submit-a-ticket).
 
-Need a helping hand? No worries. Just <a href="https://kb.jetpackcrm.com/submit-a-ticket">ask here</a>.
+### 💸 Manage your business effectively
 
+Keep your business information in order. Make proposals and send them to your contacts as quotes. Make and send invoices to your clients and get paid through the client portal.
 
+Our billing system will allow you to track transactions and payments that your clients make for your products and services (simple, effective accounting).
 
-## 💸 Manage your business effectively
+## Supercharging Jetpack CRM with optional extensions
 
-<p>Keep your business information in order. Make proposals and send them to your contacts as quotes. Make and send invoices to your clients and get paid through the client portal.</p>
+We've built a few extras for Jetpack CRM, but they're mostly offering functionality outside the core CRM, so we've separated them into individual extensions. These all have a supercharging effect on your CRM, but none are essential.
 
-<p>Our billing system will allow you to track transactions and payments that your clients make for your products and services, (simple, effective accounting).</p>
+#### Here's a selection:
+* [CRM Sales Dashboard](https://jetpackcrm.com/product/sales-dashboard/)
+* [Automations](https://jetpackcrm.com/product/automations/)
+* [Mail Campaigns](https://jetpackcrm.com/mail-campaigns/)
+* [CSV Importer Pro](https://jetpackcrm.com/product/csv-importer-pro/)
+* [Advanced Segments](https://jetpackcrm.com/product/advanced-segments/)
+* [Client Portal Pro](https://jetpackcrm.com/product/client-portal-pro/)
+* [Invoicing Pro](https://jetpackcrm.com/product/invoicing-pro/)
+* [Gravity Forms](https://jetpackcrm.com/product/gravity-forms/ (Contact forms))
+* [Contact Form 7](https://jetpackcrm.com/product/contact-form-7/ (Contact forms))
+* [AWeber](https://jetpackcrm.com/product/aweber-connect/ (Email marketing))
+* [ConvertKit](https://jetpackcrm.com/product/convertkit/ (Email marketing))
+* [PayPal Sync](https://jetpackcrm.com/product/paypal-sync/)
+* [Twilio](https://jetpackcrm.com/product/twilio/ (for SMS Sending))
+* [Plus many more...](https://jetpackcrm.com/extensions/)
```

Comment:
Could probably delete this line too, since it's linked to just below.

Reply:
Done! 92c3dcb054